src/exodus/rpctx.cpp

```diff
@@ -1528,6 +1529,90 @@ UniValue exodus_sendcreatedenomination(const UniValue& params, bool fHelp)
+            "1. \"fromaddress\"                (string, required) the address to send from\n"
+            "2. propertyid                   (number, required) the property to create mints\n"
+            "3. \"denominations\"              (string, required) A json object with denomination and amount\n"
+            "    {\n"
+            "      \"denomination\": amount    (number) The denomination id, the amount of mints\n"
+            "      ,...\n"
+            "    }\n"
+            "\nResult:\n"
+            "\"hash\"                          (string) the hex-encoded transaction hash\n"
+            "\nExamples:\n"
+            + HelpExampleCli("exodus_sendmint", "\"3M9qvHKtgARhqcMtM5cRT9VaiDJ5PSfQGY\" 1 \"{\"0\": 1, \"1\": 2}\"")
+            + HelpExampleRpc("exodus_sendmint", "\"3M9qvHKtgARhqcMtM5cRT9VaiDJ5PSfQGY\", 1, \"{\"0\": 1, \"1\": 2}\"")
+        );
+    }
+
+    // obtain parameters & info
+    std::string fromAddress = ParseAddress(params[0]);
+    uint32_t propertyId = ParsePropertyId(params[1]);
+    UniValue denominations = params[2].get_obj();
+
+    // perform checks
+    RequireExistingProperty(propertyId);
+    auto keys = denominations.getKeys();
+
+    // collect all mints need to be created
+    std::vector<uint8_t> denoms;
+    for (const auto& denom : keys) {
+        auto denomId = std::stoul(denom);
+        if (denomId < 0 || denomId > UINT8_MAX) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "invalid denomination");
+        }
+
+        auto amount = denominations[denom].get_int();
+        if (amount < 0 || amount > UINT8_MAX) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "invalid amount of mints");
+        }
+
+        denoms.insert(denoms.end(),
+            static_cast<size_t>(amount), static_cast<uint8_t>(denomId));
+    }
+
+    int64_t amount;
+    try {
+        amount = GetDenominationsSum(propertyId, denoms);
+    } catch (const std::invalid_argument& e) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, e.what());
+    }
+
+    RequireBalance(fromAddress, propertyId, amount);
+
+    std::vector<std::pair<uint8_t, exodus::SigmaPublicKey>> mints;
+    mints.reserve(denoms.size());
+    {
+        LOCK(pwalletMain->cs_wallet);
+        exodus::Wallet wallet(pwalletMain->strWalletFile);
+        wallet.GenerateMints(propertyId, denoms.begin(), denoms.end(), std::back_inserter(mints));
+    }
+
+    std::vector<unsigned char> payload = CreatePayload_SimpleMint(propertyId, mints);
+
+    // request the wallet build the transaction (and if needed commit it)
+    uint256 txid;
+    std::string rawHex;
+    int result = WalletTxBuilder(fromAddress, "", "", 0, payload, txid, rawHex, autoCommit);
+
+    // check error and return the txid (or raw hex depending on autocommit)
+    if (result != 0) {
+        throw JSONRPCError(result, error_str(result));
+    } else {
+        if (!autoCommit) {
+            return rawHex;
+        } else {
+            return txid.GetHex();
```

Comment:
According to `exodus_send`, seem like we need to invoke `PendingAdd` before this line.